vxquery-cli/src/main/java/org/apache/vxquery/cli/VXQuery.java

```diff
@@ -305,7 +305,7 @@ public void startLocalHyracks() throws Exception {
             ncConfig.dataIPAddress = localAddress;
             ncConfig.resultIPAddress = localAddress;
             ncConfig.nodeId = "nc" + (i + 1);
-            ncConfig.ioDevices = Files.createTempDirectory(ncConfig.nodeId).toString();
+            ncConfig.ioDevices = "target/tmp/indexFolder";
```

Comment:
For now let's leave this as it was. Not sure if the change would work in all scenarios